conf/application.conf

```diff
@@ -161,7 +161,7 @@ features {
   exportTiffMaxVolumeMVx = 1024
   exportTiffMaxEdgeLengthVx = 8192
   defaultToLegacyBindings = false
-  editableMappingsEnabled = false
+  editableMappingsEnabled = true
```

Comment:
unrelated to this pr. I talked to @normanrz and we agreed to include this here. will be disabled selectively in docker.